src/main/java/seedu/command/event/ListEvent.java

```diff
@@ -0,0 +1,19 @@
+package seedu.command.event;
+
+import seedu.command.Command;
+import seedu.event.EventList;
+
+public class ListEvent extends Command {
+    private EventList eventList;
+
+    public ListEvent(EventList eventList) {
+        this.eventList = eventList;
+    }
+
+    @Override
+    public void execute() {
+        for (int i = 0; i < eventList.getSize(); i++) {
+            eventList.find(i);
+        }
```

Comment:
Can try to use Anqi's UI to print nice nice, but not sure if you have time to implement that before Tuesday.